src/components/MainSwiper/MainSwiper.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+import { FreeMode, Navigation } from 'swiper/modules';
+import { ProductCard } from '@/components/ProductCard';
+//
+// import photo from '../../../public/img/banner-tablets.png'
+
+// import 'swiper/scss';
+// import 'swiper/css/free-mode';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error
+// import 'swiper/scss/navigation';
+
+// import 'swiper/css';
+// import 'swiper/scss/navigation';
+
+import 'swiper/css/bundle';
+
+import styles from './MainSwiper.module.scss';
+import { SwiperButton } from '@/components/SwiperButton';
+import { TypePhone } from '@/types/TypePhone.ts';
+
+interface Props {
+  swiperTitle: string;
+  products: TypePhone[];
+}
+
+export const MainSwiper: React.FC<Props> = ({ swiperTitle, products }) => {
+  const [reachFirst, setReachFirst] = useState(true);
+  const [reachLast, setReachLast] = useState(false);
+
+  return (
+    <>
+      <Swiper
+        slidesPerView="auto"
+        spaceBetween={16}
+        freeMode={false}
+        onReachBeginning={() => setReachFirst(true)}
+        onReachEnd={() => setReachLast(true)}
+        onFromEdge={() => {
+          setReachLast(false);
+          setReachFirst(false);
+        }}
+        breakpoints={{
+          1200: {
+            slidesPerView: 4,
+          },
+          630: {},
+          320: {},
```

Comment:
Maybe you forgot to add all breakpoints, so in a small screen slide looks not very well 
![image](https://github.com/user-attachments/assets/d55d7790-ea36-47dc-aede-4f2973d5bce1)